custom/enikshay/integrations/nikshay/repeaters.py

```diff
@@ -53,10 +125,48 @@ def episode_pending_registration_changed(case):
     return value_changed
 
 
+def person_hiv_status_changed(case):
+    last_case_action = case.actions[-1]
+    if last_case_action.is_case_create:
+        return False
+
+    last_update_actions = [update.get_update_action() for update in get_case_updates(last_case_action.form)]
+    value_changed = any(
+        action for action in last_update_actions
+        if isinstance(action, CaseUpdateAction) and 'hiv_status' in action.dynamic_properties
+    )
+    return value_changed
+
+
+def date_tested_added_for_test(case):
+    last_case_action = case.actions[-1]
+    if last_case_action.is_case_create:
+        return False
+
+    last_update_actions = [update.get_update_action() for update in get_case_updates(last_case_action.form)]
+    value_changed = any(
+        action for action in last_update_actions
+        if isinstance(action, CaseUpdateAction) and 'date_tested' in action.dynamic_properties
+    )
+    return value_changed
+
+
 def create_case_repeat_records(sender, case, **kwargs):
     create_repeat_records(NikshayRegisterPatientRepeater, case)
 
+
+def create_followup_repeat_records(sender, case, **kwargs):
+    create_repeat_records(NikshayFollowupRepeater, case)
+
+
+def create_hiv_test_repeat_records(sender, case, **kwargs):
+    create_repeat_records(NikshayHIVTestRepeater, case)
+
 case_post_save.connect(create_case_repeat_records, CommCareCaseSQL)
+case_post_save.connect(create_followup_repeat_records, CommCareCaseSQL)
+case_post_save.connect(create_hiv_test_repeat_records, CommCareCaseSQL)
 
 # TODO: Remove this when eNikshay gets migrated to SQL
 case_post_save.connect(create_case_repeat_records, CommCareCase)
+case_post_save.connect(create_followup_repeat_records, CommCareCaseSQL)
```

Comment:
this is going to create 2 repeat records for each case. Did you mean `CommCareCase` here?